app/routes/formbuilder/new.js

```diff
@@ -5,13 +5,15 @@ export default class FormbuilderNewRoute extends Route {
   @service router;
   @service store;
 
-  async model() {
+  async model(params, transition) {
```

Comment:
If we go the custom component way, I think we can just remove this route altogether. The creation and redirecting could also be part of that component.

It feels a bit strange to have validation logic in a modal while it's still possible to sidestep it by going to the route directly. So I think keeping this together would be "better".

Reply:
As mentioned in the other comment, moving this to the index controller keeps things together and makes this route unnecessary.